README.md

```diff
@@ -40,227 +40,8 @@ python manage.py test mod_app/tests
 
 ---
```

Comment:
I think you should include setting up the virtual environment and running migrations; e.g.:

After cloning this repo, create a new virtual environment and install the required python packages:
```
python3 -m venv modvenv
source modvenv/bin/activate
pip install -r requirements-base.txt
```
You can then run a local instance:
```
python manage.py migrate
python manage.py runserver
```